src/templates/LectureView/index.js

```diff
@@ -23,6 +31,55 @@ const LectureView = ({ data }) => {
           />
         </div>
         <div>
+          <div className=" px-4 py-3 mb-3 flex flex-wrap items-center justify-between bg-gray-900">
+            <div className="w-full flex justify-between lg:w-auto  pl-4  lg:block lg:justify-start">
+              <Link
+                className="text-sm font-bold leading-relaxed inline-block mr-4 py-2 whitespace-no-wrap text-white hover:opacity-75"
+                to={`/courses/${strapiCourse.slug}`}
+              >
+                {courseTitle}
+              </Link>
+            </div>
+            <div className="lg:flex flex-grow items-center">
+              <div className="flex flex-col lg:flex-row ml-auto">
+                <div className="px-3 py-2  text-xs font-bold leading-snug text-white ">
+                  {title}
+                </div>
+                <div className="flex">
+                  {position === strapiCourse.lectures.length ? (
```

Comment:
maybe put `position === strapiCourse.lectures.length` in a variable called `isLastLecture` or something, to make the code easier to understand .. same for `position===1`

Reply:
done